### PR TITLE
Rebeka: Happy thoughts project

### PR DESCRIPTION
**Pull request by rebkova**
Link to my deployed project: https://rebeka-happy-thoughts.netlify.app/

I got stuck with making the like button to work. Otherwise everything went rather smooth. I really enjoyed the project and am enjoying working with React a lot. There is still so much to grasp but I'll get there eventually. :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,26 @@
 # Happy Thoughts
+A Twitter like app with positive vibes. :sunny:
```
Comment (BamStugendal) [inline README.md:2]: Very good README, very clear and informative! 😄

File: code/src/urls.js [hunk 2]
```diff
@@ -0,0 +1 @@
+export const THOUGHTS_URL = "https://happy-thoughts-technigo.herokuapp.com/thoughts";
```
Comment (BamStugendal) [inline code/src/urls.js:1]: Nice that you did a separate file for URL 👍

File: code/src/components/ThoughtInput.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            aria-label="heart"
+            role="img">
+            &#10084;&#65039;
+        </span>
+          {" "}Send Happy Thought{" "}
+          <span
+            aria-label="heart"
+            role="img">
+            &#10084;&#65039;
+        </span>
```
Comment (BamStugendal) [inline code/src/components/ThoughtInput.js:41]: Be consistent in how you separate lines. In the ThoughtList component, you separate lines different 😃

File: code/src/components/thoughts-list.css [hunk 4]
```diff
@@ -0,0 +1,49 @@
+@import url('https://fonts.googleapis.com/css2?family=Inconsolata:wght@500&display=swap');
+.thoughts-list-container {
+  font-family: 'Inconsolata', monospace;
+  display: flex;
+  justify-content: center;
+}
+
+.thought-container {
+  border: 2px solid #000000;
+  background-color: #FFFFFF;
```
Comment (BamStugendal) [inline code/src/components/thoughts-list.css:10]: You can shorten the HEX color code for white to `#fff` and black to `#000`. Less to write 😉

Comment (BamStugendal) [inline code/src/components/thoughts-list.css:10]: You can shorten HEX color code for white to `#fff` and black to `#000`. Less to write 😉

File: code/src/App.js [hunk 5]
```diff
@@ -1,9 +1,59 @@
...
+        <ThoughtsList
+          key={thought._id}
+          thought={thought}
+          happyThought={thought.message}
+          timeStamp={thought.createdAt}
+          nrOfLikes={thought.hearts}
+          onLike={handleLike}
+        />
+      ))}
     </div>
```
Comment (BamStugendal) [inline code/src/App.js:57]: Maybe make `<div className="app-container">` to `<section className="app-container">` to make it more semantic? 😄

**Review by BamStugendal — APPROVED** [review]
Good job Rebeka! Very structured code and a good README, liked that you made a bullet list there. I think I will take inspiration from you and do that for my next project 😌

**Comment by rebkova** [comment]
Thanks for the explanation regarding s`etInterval()`, Maks. In my simple world I thought I could use it as I did. :) I've removed the line of code now and added `id` to the` input` tag.

**Pull request closed without merging**